scripts/format_code.sh

```diff
@@ -0,0 +1,6 @@
+#! /bin/sh
+
+for file in $(git ls-files | grep '\.[hc]$' | grep -v '^cios-lib')
```

Comment:
We can add the files/paths we want to ignore to the `.clang-format-ignore` file (example: [.clang-format-ignore](https://github.com/xerpi/fakemote/blob/main/.clang-format-ignore)).